app/src/main/java/me/tatarka/bindingcollectionadapter/sample/FragmentListView.java

```diff
@@ -20,7 +22,14 @@
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setRetainInstance(true);
-        viewModel = new ViewModel(true);
+
+        ObservableList<ItemViewModel> items = new ObservableArrayList<>();
+        items.add(new HeaderItemViewModel("我是header"));
```

Comment:
Hard-coded strings are not a good idea.... If you must put a string somewhere, you should use the `strings.xml` resource file (which would also allow localization; unfortunately not everybody reads Hanzi...)